pootle/apps/pootle_statistics/models.py

```diff
@@ -554,33 +532,10 @@ def get_paid_wordcounts(self):
         translated_words = round(translated_words, 4)
         reviewed_words = ns
 
-        def get_sugg_reviewed_accepted():
-            suggester = self.submission.suggestion.user.pk
-            reviewer = self.submission.submitter.pk
-            if suggester == reviewer:
-                if self.submission.old_value == '':
-                    return translated_words, None
-            else:
-                return None, reviewed_words
-
-            return None, None
-
-        def get_sugg_accepted():
-            suggester = self.submission.suggestion.user.pk
-            reviewer = self.submission.submitter.pk
-            if suggester != reviewer and self.submission.old_value == '':
-                return translated_words, None
-
-            return None, None
-
         def get_edited():
             return None, reviewed_words
 
         return {
             TranslationActionCodes.NEW: lambda: (translated_words, None),
             TranslationActionCodes.EDITED: get_edited,
-            TranslationActionCodes.REVIEWED: lambda: (None, reviewed_words),
```

Comment:
as i think it wouldnt work any longer as it did - and not thinking we will need it again